Twig/Extension.php

```diff
@@ -118,7 +127,11 @@ private function addDefaultApiCalls()
                 return; // Do not add 'trackPageView'
             }
         }
-
-        $this->paqs[] = ['trackPageView'];
+        if (!empty($lastPaq)) {
```

Comment:
If you initialize with `null`, you can change this check to `null !== $manuallyAddedTrackPageView`